jcl/source/windows/JclSvcCtrl.pas

```diff
@@ -1086,7 +1086,7 @@ procedure TJclSCManager.Refresh(const RefreshAll: Boolean);
       repeat
         ReallocMem(PBuf, BytesNeeded);
         ServicesReturned := 0;
-        Ret := EnumServicesStatus(FHandle, SERVICE_TYPE_ALL, SERVICE_STATE_ALL,
+        Ret := EnumServicesStatus(FHandle, SERVICE_WIN32 or SERVICE_ADAPTER or SERVICE_DRIVER or SERVICE_INTERACTIVE_PROCESS, SERVICE_STATE_ALL,
```

Comment:
The cause of this error on a change in SERVICE_TYPE_ALL constant on Delphi unit Winapi.WinSvc starting from 10.4:

  SERVICE_TYPE_ALL = (SERVICE_WIN32 or
                                   SERVICE_ADAPTER or
                                   SERVICE_DRIVER or
                                   SERVICE_INTERACTIVE_PROCESS or
                                   SERVICE_USER_SERVICE or
                                   SERVICE_USERSERVICE_INSTANCE or
                                   SERVICE_PKG_SERVICE);

in previous Delphi 10.3 this constant was

  SERVICE_TYPE_ALL = (SERVICE_WIN32 or
                                  SERVICE_ADAPTER or
                                  SERVICE_DRIVER or
                                  SERVICE_INTERACTIVE_PROCESS);


TjsclManager.Refresh called the EnumServicesStatus api in this way:

        Ret := EnumServicesStatus(FHandle, SERVICE_TYPE_ALL , SERVICE_STATE_ALL,
          PEnumServiceStatus(PBuf){$IFNDEF FPC}{$IFNDEF RTL340_UP}^{$ENDIF}{$ENDIF},
          BytesNeeded, BytesNeeded, ServicesReturned, ResumeHandle);

 this causes the problem in older sysop because as you can see here:

https://docs.microsoft.com/en-us/windows/win32/api/winsvc/nf-winsvc-enumservicesstatusexa

EnumServicesStatus allows only a few of the declared constants for the dwServiceType parameter.